### PR TITLE
Draft content for Conversations Section

### PR DESCRIPTION
**Pull request by AnthonyRollins**

### CODE REVIEW

File: docs/conversations/README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# conversations/README.md
+# Conversations
+
+The Conversations tab is our shared inbox for emails coming and going from crm@calitp.org and hello@calitp.org. Anyone with access to Conversations can respond to inquiries using these emails, and the email will come from the same sender, “Cal-ITP” no matter who responds.
```
Comment (laneymangan) [inline docs/conversations/README.md:3]:
```suggestion
The Conversations tab is our shared inbox for emails coming and going from crm@calitp.org and hello@calitp.org. Anyone with access to Conversations can respond to inquiries using these emails, and the email will come from the same sender, “Cal-ITP,"  no matter who responds.
```

File: docs/conversations/conversations.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Conversations
+
+The Conversations tab is our shared inbox for emails coming and going from crm@calitp.org  and hello@calitp.org. Anyone with access to Conversations can respond to inquiries using these emails, and the email will come from the same sender, “Cal-ITP” no matter who responds.
+
+**How the shared inbox works:**
+While the inbox is shared, your view of your inbox is personal, meaning, if you read messages, they won’t show up as unread in the inbox of other team members. This also includes when you send an email from the inbox, the sender will say Cal-ITP, and will show up as a new message in other team members’ inboxes while yours would be read and sent. Conversations are a separate process from [email logging](/docs/getting-started/email-setup.md), where emails can be sent/recorded from your personal email.
```
Comment (laneymangan) [inline docs/conversations/conversations.md:6]:
```suggestion
While the inbox is shared, your view of your inbox is personal, meaning, if you read messages, they won’t show up as unread in the inbox of other team members. This includes when you send an email from the inbox: the sender will say "Cal-ITP" and will show up as a new message in other team members’ inboxes while yours would be read and sent. Conversations are a separate process from [email logging](/docs/getting-started/email-setup.md), where emails can be sent/recorded from your personal email.
```

File: docs/conversations/add-a-comment.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+!!! Note
+
+    Utilizing the Comment tool is a great way to communicate with your team internally without involving external parties edits
+
+**2. Click "Conversations"**
+
+![](https://image.scribehow-prod.com/d5qAb-VWTDht_RT_lw3UmS4jhAAQ-VMLiddhIxQGhHU/zoom:0.3750837240455459/enlarge:true/crop:1493:840:nowe:0:0/wm:0.8:nowe:144:-3:0.17857142857142858/aHR0cHM6Ly9jb2xvbnktcmVjb3JkZXIuczMuYW1hem9uYXdzLmNvbS9maWxlcy8yMDIyLTA4LTE5LzRkYzRkZDY5LWMyZDAtNDc2My1hNTBiLTUyZDFjNmM1MDk0ZC9hc2NyZWVuc2hvdC5qcGVn)
+
+**3. Click "Inbox"**
```
Comment (thekaveman) [inline docs/conversations/add-a-comment.md:13]:
```suggestion
**3. Click "Outbox"**
```

Comment (thekaveman) [reply docs/conversations/add-a-comment.md:13]: This was just a sample!

**Pull request merged**